addons/csw/functions/fnc_reload_handleAddTurretMag.sqf

```diff
@@ -0,0 +1,56 @@
+/*
+ * Author: TCVM, PabstMirror
+ * Handles adding ammo to a turret
+ * Called from a global event but only runs where turret is local
+ *
+ * Arguments:
+ * 0: Static Weapon <OBJECT>
+ * 1: Turret Path <ARRAY>
+ * 2: Unit doing action <OBJECT>
+ * 3: Vehicle Magazine <STRING>
+ * 4: Ammo in magazine <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorTarget, [0], player, "200Rnd_127x99_mag_Tracer_Red", 70] call ace_csw_fnc_reload_handleAddTurretMag
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_turret", "_unit", "_carryMag" ,"_ammoRecieved"];
+TRACE_5("reload_handleAddTurretMag",_vehicle,_turret,_unit,_carryMag,_ammoRecieved);
+
+TRACE_2("",local _vehicle, _vehicle turretLocal _turret);
+if (!(_vehicle turretLocal _turret)) exitWith {};
+
+([_vehicle, _turret, _carryMag] call FUNC(reload_canLoadMagazine)) params ["_canAdd", "_loadedMag", "_neededAmmo", "_isBeltLinking"];
+TRACE_4("canLoad",_canAdd,_loadedMag,_neededAmmo,_isBeltLinking);
+
+private _ammoRemaining = _ammoRecieved;
+if (_canAdd) then {
+    private _ammoUsed = _neededAmmo min _ammoRecieved;
+    _ammoRemaining = _ammoRemaining - _ammoUsed;
+
+    if (_isBeltLinking) then {
+        private _currentAmmo = _vehicle magazineTurretAmmo [_loadedMag, _turret];
+        _currentAmmo = _currentAmmo + _ammoUsed;
+        TRACE_2("Setting mag ammo",_loadedMag,_currentAmmo);
+        _vehicle setMagazineTurretAmmo [_loadedMag, _currentAmmo, _turret];
```

Comment:
`setMagazineTurretAmmo` broken by BIS.
It is not working if turret **and** vehicle not local to issuer.
Because this is **CREW** served weapons system it must allow to link and load ammo by assistant player while other player sitting on gunner slot.
There is linking code in ace_reload module. Maybe it will help.
Also if CSW will be applieble to vehicles `setMagazineTurretAmmo` also will cause problems because vehicle is local to driver and not gunner.

Reply:
seems like the normal `setAmmo` command will work
https://github.com/acemod/ACE3/pull/5652/commits/386d6aec75e4be9fe919037cca1632a36a44c871#diff-f2d09afb4c14c8d9fc63da546b5a0d9bR46

I had planned to expand this to cover full vehicles, (like m2 turrets on tanks) at some point and I'm worried how this will work with multi turreted vehicles
but it seems to work just fine for static weapons
and, it's really only required for belt linking